ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -4375,4 +4375,65 @@ EC2SC:
   - supply_air_temperature_sensor
   - supply_air_temperature_setpoint
   implements:
-  - CONTROL
\ No newline at end of file
+  - CONTROL
+
+CHWISVPCM:
+  description: "Chllled water isolation valve percentage control and monitoring"
+  is_abstract: true
+  implements:
+  - OPERATIONAL
+  opt_uses:
+  - chilled_water_flowrate_alarm
+  - chilled_water_isolation_valve_mode
+  - chilled_water_isolation_valve_percentage_setpoint
+  uses:
+  - chilled_water_isolation_valve_percentage_command
+  - chilled_water_isolation_valve_percentage_sensor
+
+CDWISVPCM:
+  description: "Condensing water isolation valve percentage control and monitoring"
+  is_abstract: true
+  implements:
+  - OPERATIONAL
+  opt_uses:
+  - condensing_water_flowrate_alarm
+  - condensing_water_isolation_valve_mode
+  - condensing_water_isolation_valve_percentage_setpoint
+  uses:
+  - condensing_water_isolation_valve_percentage_command
+  - condensing_water_isolation_valve_percentage_sensor
+
+PSMC:
```

Comment:
It is for the chill water pump 
The pump_speed_mode is changing the mode between auto and manual, i will change to control mode
The pump_speed_frequency_setpoint is to the user input to set the 0-50Hz which is the speed of pump

Reply:
you will model chilled water pumps on their own (they are not integral to a device, so you dont need to use the `pump` subfield. We know its on a pump because its the run command on a pump entity.

They type you are describing seems to be [this](https://github.com/google/digitalbuildings/blob/7c1c952dc9a253ee5abf190bb98c097c89757471/ontology/yaml/resources/HVAC/entity_types/PMP.yaml#L56). Please check and adjust.

Reply:
The PSMC abstract type has been removed and the PMP types are updated